### DISCUSSION TITLE
Schism Tracker 20240630

### DISCUSSION THREAD

**Discussion by mrpapersonic** [schismtracker/schismtracker#486] [Announcements]
Summary of changes since the [20240614](https://github.com/schismtracker/schismtracker/releases/tag/20240614) release:
* Fix a crash in the sample library under macOS's standard C library
* Allow loading of any formats supported by Microsoft's Media Foundation API (requires Windows 7 or newer)
* Schism now runs on Windows XP again
* Add DSM format loader (thanks @RepellantMold)
* Fix some filenames being sorted incorrectly
* Fix a crash upon loading some IT modules made in Modplug and its derivatives
* **EXPERIMENTAL**: Add XI instrument saving

<hr /><em>This discussion was created from the release <a href='https://github.com/schismtracker/schismtracker/releases/tag/20240630'>Schism Tracker 20240630</a>.</em>

**Comment by Iskmarr** [comment]
The KEYPAD doesn't work properly anymore on MacOS

**Reply by esaruoho** [reply]
@Iskmarr please write a ticket about this here: https://github.com/schismtracker/schismtracker/issues/new
